MMUL-BASH-01/Exercises/Day-4/Exercise.004.solutions

```diff
@@ -0,0 +1,23 @@
+# Exercise 004 - Cycle through all files in a dir and print details
+
+1) Cycle through files beginning with 'h' in the dir '/etc' and print, for each file,
```

Comment:
Do not repeat the text of the question, comment it out on how it is solved in the solution.